test/core/CMakeLists.txt

```diff
@@ -26,5 +26,9 @@ add_subdirectory(sector_storage)
 add_subdirectory(sectorblocks)
 add_subdirectory(serialization)
 add_subdirectory(storage)
-add_subdirectory(test_vectors)
+
+if (TESTING_PROOFS)
```

Comment:
Looks like TESTING_ACTORS is more suitable

Reply:
Name has been changed.